rust/kernel/print.rs

```diff
@@ -233,6 +339,35 @@ macro_rules! pr_emerg (
     )
 );
 
+/// Prints an emergency-level message (level 0) with ratelimit.
+///
+/// Use this level if the system is unusable.
+///
+/// Equivalent to the kernel's [`pr_emerg_ratelimited`] macro.
+///
+/// Mimics the interface of [`std::print!`]. See [`core::fmt`] and
+/// [`alloc::format!`] for information about the formatting syntax.
+///
+/// [`pr_emerg`]: https://www.kernel.org/doc/html/latest/core-api/printk-basics.html#c.pr_emerg
+/// [`std::print!`]: https://doc.rust-lang.org/std/macro.print.html
+///
+/// [`where`] parameter is a string literal that will be used as an
```

Comment:
`where` cannot be a link (`rustdoc` warns); same for the others:

```suggestion
/// `where` parameter is a string literal that will be used as an
```